packages/icons-react16/scripts/constants.ts

```diff
@@ -4,7 +4,8 @@ export const WarningWeakIconColor = "#E57723"; // --hop-warning-icon-weak
 
 export const DecorativeOption7IconColor = "#2A2620"; // --hop-decorative-option7-icon
 export const DecorativeOption7SurfaceColor = "#E5DED6"; // --hop-decorative-option7-surface
-export const White = "#fff"; // --hop-status-neutral-surface
+export const WhiteHexadecimal = "#fff";
```

Comment:
Unfortunately not, because we're testing whether the color `#fff `is used in the svg to replace it with the variable. If we set this value to `#FFFFFF` the test returns false and the color will not be replaced.